vm/factory/systemSCFactory.go

```diff
@@ -249,6 +249,11 @@ func (scf *systemSCFactory) createGovernanceContract() (vm.SystemSmartContract,
 }
 
 func (scf *systemSCFactory) createDelegationContract() (vm.SystemSmartContract, error) {
+	addTokensAddress, err := scf.addressPubKeyConverter.Decode(scf.systemSCConfig.DelegationManagerSystemSCConfig.ConfigChangeAddress)
```

Comment:
duplicated L278-L281. Could have been another function that computed these bytes and provide them to createDelegationContract and createDelegationManagerContract

Reply:
would leave as it is.